src/controllers/users.controller.ts

```diff
@@ -29,3 +29,35 @@ export const getUsersController = async (): Promise<ISaveUserResponse[]> => {
         throw new Error(message);
     }
 };
+
+export const getUserController = async (user: IUser): Promise<IUser> => {
+    try {
+        const { id } = user;
+        const { data } = await transportLayer.get(`/users/${id}`);
+        return data;
+    } catch (err) {
+        const { message } = err.data;
+        throw new Error(message);
+    }
+};
+
+export const deleteUserController = async (user: any): Promise<any> => {
```

Comment:
For this API call you need only user ID, you shouldn't provide fully user object in this function. You need only id